features/docs/integration/gegeven-stap-definities-persoon.feature

```diff
@@ -0,0 +1,42 @@
+  #language: nl
+
+@integratie @stap-documentatie
+Functionaliteit: Persoon stap definities
+
+  Achtergrond:
+    Gegeven de tabellen 'lo3_pl' bevat geen rijen
+
+  Scenario: de persoon met burgerservicenummer '[bsn]'
+    Gegeven de persoon 'Gerda' met burgerservicenummer '000000012'
+    Als de sql statements gegenereerd uit de gegeven stappen zijn uitgevoerd
+    Dan heeft de persoon 'Gerda' de volgende rij in tabel 'lo3_pl'
+    | pl_id | geheim_ind |
+    | 1     | 0          |
+    En heeft de persoon 'Gerda' de volgende rij in tabel 'lo3_pl_persoon'
+    | pl_id | persoon_type | stapel_nr | volg_nr | burger_service_nr |
```

Comment:
graag ook geslachts_naam vullen met de identificatie van de persoon, in dit voorbeeld 'Gerda'

Reply:
wijziging is doorgevoerd